airbyte-integrations/connectors/source-jdbc/build.gradle

```diff
@@ -25,6 +25,9 @@ dependencies {
     implementation project(':airbyte-integrations:bases:base-java')
     implementation project(':airbyte-protocol:models')
 
+    implementation 'io.debezium:debezium-embedded:1.4.2.Final'
+    implementation 'io.debezium:debezium-api:1.4.2.Final'
+    implementation 'io.debezium:debezium-connector-postgres:1.4.2.Final'
```

Comment:
This doesn't feel right to be at the `source-jdbc` level

Reply:
i might have moved too many deps. i suspect just the first of these 3 deps is needed. the reason they are here is that state management stuff all live in source-jdbc. does that make you feel better or not really? if i split the loading and reading the state from the debezium file out i might be able to lance these dependencies. i think it might introduce some other problems, but i can try it!

Reply:
I'm specifically concerned about `io.debezium:debezium-connector-postgres:1.4.2.Final`, not the others. I just don't want postgres-related dependencies for all jdbc sources.

Reply:
fixed it.

Reply:
100% agree. postgres there was a copy pasta.

managed to get all of the debezium stuff out of there and that fixed all the path issues. turned out better!